eng/tox/tox.ini

```diff
@@ -130,4 +130,24 @@ deps =
   {[tools]deps}
 commands = 
     {envbindir}/python {toxinidir}/../../../eng/tox/create_wheel_and_install.py -d {distdir} -p {toxinidir}
-    {envbindir}/python {toxinidir}/../../../eng/tox/import_all.py -t {toxinidir}
\ No newline at end of file
+    {envbindir}/python {toxinidir}/../../../eng/tox/import_all.py -t {toxinidir}
+
+
+[testenv:verifywhl]
+skipsdist = true
+skip_install = true
+changedir = {toxinidir}
+deps = 
+commands =
+    {envbindir}/python {toxinidir}/../../../eng/tox/create_wheel_and_install.py -d {distdir} -p {toxinidir} --skip-install True
+    {envbindir}/python {toxinidir}/../../../eng/tox/verify_whl.py -d {distdir} -t {toxinidir}
+
+
+[testenv:verifysdist]
+skipsdist = true
+skip_install = true
```

Comment:
I believe you briefly mentioned this last week. We are manually building the sdist on line 152. Is there anything preventing you from leaving `skipsdist = false` while having `skip_install = false`?

If so, then you won't need to have line 152 at all. However, I believe I recall you mentioning that it goes against expectations and installs regardless. I'd like to see that though.